views/prisonerProfile/prisonerFinance/privateCash.njk

```diff
@@ -33,8 +33,7 @@
     {{ govukTable({
       head: [
         { text: "Date" },
-        { text: "Money in" },
-        { text: "Money out" },
+        { text: "Amount" },
```

Comment:
Isn't this reliant on `mergeAmounts` being `true`?

Reply:
It's always true for that section. Having a central mapper that works for difference screens is a bit of a smell here and is causing confusion.